pkg/v1/google/keychain.go

```diff
@@ -77,5 +77,5 @@ func resolve() (authn.Authenticator, error) {
 		return auth, nil
 	}
 
-	return nil, fmt.Errorf("failed to create token source from env: %v or gcloud: %v", envErr, gErr)
+	return nil, fmt.Errorf("failed to create token source from env: %v or gcloud: %v", envErr, gErr) //nolint: errorlint
```

Comment:
I am curious why you chose to not wrap here?

Reply:
Ah the long line hid that.  I don't especially care since this seems like a relatively niche part of the library.